be/src/exprs/vectorized/runtime_filter_bank.cpp

```diff
@@ -263,22 +264,29 @@ void RuntimeFilterProbeCollector::do_evaluate(vectorized::Chunk* chunk, RuntimeB
         return;
     }
     if (!eval_context.selectivity.empty()) {
+        const auto num_rows = chunk->num_rows();
+        auto& selection = eval_context.running_context.selection;
+        size_t true_count = 0;
+        selection.assign(num_rows, 1);
         for (auto& kv : eval_context.selectivity) {
             RuntimeFilterProbeDescriptor* rf_desc = kv.second;
             const JoinRuntimeFilter* filter = rf_desc->runtime_filter();
-            if (filter == nullptr) continue;
+            if (filter == nullptr) {
+                continue;
+            }
             ColumnPtr column = rf_desc->probe_expr_ctx()->evaluate(chunk);
-            vectorized::Column::Filter& selection = filter->evaluate(column.get(), &eval_context.running_context);
+            filter->evaluate(column.get(), &eval_context.running_context);
+            true_count = SIMD::count_nonzero(selection);
```

Comment:
`filter->evaluate` has already returned `true_count`, so maybe we needn't `SIMD::count_nonzero`?
```suggestion
            true_count = filter->evaluate(column.get(), &eval_context.running_context);
```